backend/src/main/java/com/ody/notification/domain/types/AbstractNotification.java

```diff
@@ -6,26 +6,26 @@
 import com.ody.notification.domain.NotificationStatus;
 import com.ody.notification.domain.NotificationType;
 import java.time.LocalDateTime;
-import lombok.AllArgsConstructor;
 
-@AllArgsConstructor
 public abstract class AbstractNotification {
 
-    private final Long id;
     private final Mate mate;
     private final NotificationType type;
     private final LocalDateTime sendAt;
     private final NotificationStatus status;
     private final FcmTopic fcmTopic;
 
-    public AbstractNotification(
+    protected AbstractNotification(
             Mate mate,
-            NotificationType type,
             LocalDateTime sendAt,
             NotificationStatus status,
             FcmTopic fcmTopic
     ) {
-        this(null, mate, type, calculateSendAt(sendAt), status, fcmTopic);
+        this.mate = mate;
+        this.type = getType();
+        this.sendAt = calculateSendAt(sendAt);
+        this.status = status;
+        this.fcmTopic = fcmTopic;
     }
 
     private static LocalDateTime calculateSendAt(LocalDateTime sendAt) {
```

Comment:
[제안]
static 키워드 이제는 제거해주어도 되겠네요! 이 부분은 제가 코드 수정하면서 반영하겠습니다ㅎㅎ
```suggestion
    private LocalDateTime calculateSendAt(LocalDateTime sendAt) {
```